source/quick-start.txt

```diff
@@ -2,5 +2,175 @@
+  :class: note
+
+  Specifying ``-y`` tells NPM to use default values. You can omit the ``-y``
+  flag to proceed through an interactive selection process for common project
+  settings.
+
+
+
+Add MongoDB as a Dependency
+---------------------------
+
+Next, add the Node MongoDB driver as a project dependency. Use the ``--save``
+flag to save the dependency to ``package.json`` file that was automatically
+generated in the previous step.
+
+.. code-block:: sh
+
+  npm install --save mongodb
+
+NPM will download dependencies and output information to your terminal. After
+installing, you will have a ``node_modules`` directory added to your workspace.
+
+
+Create a Free Tier Cluster in Atlas
+-----------------------------------
+
+The easiest way to get started with MongoDB is by using MongoDB Atlas.
+Atlas doesn't require a credit card to sign up, and offers free-tier deployments that are free for life.
+Follow along with our :atlas:`steps to create and set up a free-tier cluster <tutorial/create-atlas-account/>`, then :atlas:`load sample data <sample-data/load-sample-data/>`
+to get the data necessary for the rest of this guide.
+
+
+Store Configuration Separately
+------------------------------
+
+A tenet of modern applications is to `store
+configuration data in the environment <https://12factor.net/config>`_.
+
+We'll use an NPM module called ``dotenv`` to help us hide our sensitive data and
+put it in an environmental variable. ``dotenv`` allows us to easily read in
+environmental variables that we define in a local file called ``.env``. We'll
+add this file to our ``.gitignore`` to ensure we don't accidentally commit our
+sensitive data, in this case our database credentials, to our version control
+system.
+
+Add ``dotenv`` to your project and create the necessary files. Enter the following
+in your terminal:
+
+.. code-block:: sh
+
+  npm install --save dotenv
+  touch .env
+  echo ".env" >> .gitignore
+  echo "MONGODB_URI=<your_atlas_uri>" >> .env
+
+
+
+Add your Atlas Connection String
+--------------------------------
+
+Open your newly created ``.env`` file. You may have to set your system to display
+hidden files. Once your ``.env`` is open, copy and paste your connection string
+from Atlas, replacing ``<your_atlas_uri>``. Ensure you update ``<username>`` and
+``<password>`` in your connection string.
+
+Connect to your Cluster
+-----------------------
+
+The final step is to create a file in your project directory called ``index.js``.
+
+.. code-block:: sh
+
+  touch index.js
+
+Open this file in your preferred editor, such as `Visual Studio Code <https://code.visualstudio.com/Download>`_, and
+paste in the following code.
+
+.. code-block:: js
+
+  const { MongoClient } = require("mongodb");
+  require("dotenv").config();
+
+  if (!process.env.MONGODB_URI) {
+    throw Error("MONGODB_URI not defined in a .env file. Did you set it up?");
+  }
+
+  const client = new MongoClient(process.env.MONGODB_URI, {
+    useNewUrlParser: true,
```

Comment:
Going to disagree on this since users are instructed to copy/paste their URI from atlas for the average use case which would clobber away those URI parameters. Eliminating ``useNewUrlParser`` though -- that's defaulting to true now, so it's not necessary.